src/layouts/Header/Dropdown.spec.jsx

```diff
@@ -91,6 +96,31 @@ describe('Dropdown', () => {
         )
       })
 
+      it('removes session expiry tracking key on sign out', async () => {
+        const { user } = setup()
+
+        jest.spyOn(console, 'error').mockImplementation()
+        const mockRemoveItem = jest.spyOn(
+          window.localStorage.__proto__,
+          'removeItem'
+        )
```

Comment:
Can you move this mock into the setup function, in case any new tests in the future they may end up accidentally calling it are already mocked out.